src/components/PrizePage/Prizes.js

```diff
@@ -1,9 +1,94 @@
 import React from 'react';
 import { basename } from 'path';
 import { graphql, useStaticQuery } from 'gatsby';
-import ComingSoon from '../ComingSoon/ComingSoon';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import grey from '@material-ui/core/colors/grey';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { Typography } from '@material-ui/core';
+import Img from 'gatsby-image/withIEPolyfill';
+
+import { ReactComponent as Squiggle } from './images/squiggle.svg';
+
+const prizeImageRadius = '50px';
+const allPrizes = [
+	{
+		img: 'best-overall',
+		track: 'Best Overall',
+		prize: '$240 Amazon Gift Card'
+	},
+	{
+		img: 'health-wellness',
+		track: 'Health and Wellness',
+		prize: '$120 Amazon Gift Card'
+	},
+	{
+		img: 'sustainability',
+		track: 'Sustainability',
+		prize: '$120 Amazon Gift Card'
+	},
+	{
+		img: 'education',
+		track: 'Education',
+		prize: '$120 Amazon Gift Card'
+	},
+	{
+		img: 'equity',
+		track: 'Equity',
+		prize: '$120 Amazon Gift Card'
+	},
+	{
+		img: 'you-do-you',
+		track: 'You Do You',
+		prize: '$120 Amazon Gift Card'
+	}
+];
+
+const useStyles = makeStyles({
+	prizeGrid: {
+		textAlign: 'center',
+		paddingBottom: '20px'
```

Comment:
i think we use theme.spacing(3) instead of specifying pixels for all of the spacing and padding